style.css

```diff
@@ -0,0 +1,151 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+* *::before *::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  margin: 10px;
+  padding: 10px;
+  border: 20px;
+  background-image: url(Header.svg);
+  max-height: 100vh;
+  max-width: auto;
+}
+
+a {
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 10px;
+  letter-spacing: 2px;
+  text-decoration: none;
+  background: none;
+  padding: 5px 5px;
+  display: inline-block;
+  width: 100%;
+  text-align: center;
+}
+
+a:hover {
+  background-color: #6070ff;
+}
+
+a:enabled {
+  background-color: #fff;
+  color: #7f8cff;
+}
+
+a:disabled {
+  background-color: #5e6c84;
+}
+
+a:active {
+  background: #2230d2;
+}
+
+.toggleNav {
+  display: none;
+}
+
+img {
+  width: 10%;
+  height: auto;
+}
+
+h1 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 48px;
+  font-weight: 800;
+  line-height: 60px;
+}
+
+h6 {
+  color: #7f8cff;
+  font-size: 16px;
+}
+
+.logo {
+  /* border: 1px solid orange; */
+  display: flex;
+  margin-left: 100px;
+}
+
+li {
+  list-style-type: none;
+}
+
+.header {
+  /* border: 1px solid yellow; */
+  display: flex;
+  margin: 0;
+  padding: 0;
+  justify-content: space-between;
+}
+
+.flex-nav ul {
+  /* border: 1px solid black; */
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  /* border: 1px solid green; */
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  justify-content: center;
+}
```

Comment:
- [ ] The headline section should contain a background image, which is presented on your website ✌ However, Kindly set it to the headline section instead and please ensure to fix its position and fix the duplicating issue.

## your design 
![image](https://user-images.githubusercontent.com/68030297/219696035-a88f8c76-6218-4fb5-a67f-55f1990e06eb.png)

## Figma design
![image](https://user-images.githubusercontent.com/68030297/219696246-7d82381b-e960-4efd-9f4b-a497dc695077.png)

- you can follow the example below to prevent the background from repeating 

````css
.your_class {
    background-repeat: no-repeat;
}
```

and for more in-depth information about CSS background feel free to follow [background-image](https://css-tricks.com/almanac/properties/b/background-image/)